src/graph_notebook/magics/graph_magic.py

```diff
@@ -2095,6 +2095,8 @@ def load(self, line='', local_ns: dict = None):
         parser.add_argument('--allow-empty-strings', action='store_true', default=False,
                             help='Load empty strings found in node and edge property values.')
         parser.add_argument('-n', '--nopoll', action='store_true', default=False)
+        parser.add_argument('--edge-only-load', action='store_true', default=False, 
+                            help='Load the graph edges directly')
```

Comment:
Perhaps the help should be something like - "Assume there are only edge files present, do not scan for vertex files before loading edge files found"